test/js/ez-selection-activate-element.js

```diff
@@ -0,0 +1,117 @@
+describe('ez-selection-activate-element', function() {
+    let element;
+    let elementSubsetElementChecked;
+
+    beforeEach(function () {
+        element = fixture('BasicTestFixture');
+        elementSubsetElementChecked =  fixture('TestFixtureSubsetElementChecked');
+    });
+
+    function checkAndclickOneElement(elem, selector, isChecked) {
+        const checkbox = elem.querySelector(selector);
+        // we need to prevent the click event at the document level
+        // to prevent Edge/Safari from checking the input when clicked
+        // so preventDefault() will be called in the test but we verify
+        // that's it not called, that's why `preventDefaultCalled`
+        // local variable was introduced.
+        let preventDefaultCalled = false;
+        const prevent = function(e) {
+            preventDefaultCalled = e.defaultPrevented;
+            e.preventDefault();
+        };
+
+        document.addEventListener('click', prevent);
+        checkbox.checked = isChecked;
+        checkbox.dispatchEvent(new CustomEvent('click', {
+            bubbles: true,
+        }));
+
+        assert.isFalse(preventDefaultCalled);
+    }
+
+    function checkAndclickEveryElements(elem, selector, isChecked) {
+        const checkboxes = elem.querySelectorAll(selector);
+        // same comment than in checkAndclickOneElement().
```

Comment:
when you write this kind of comment, it's usually the sign that you use the same code (workaround here) twice, so you should put a in a dedicated function...